datasets/ho/make_data.py

```diff
@@ -35,6 +38,7 @@
 
 # Train
 for subject in sorted(train_root_path.iterdir()):
+    sys.stdout.write(f"writing subject in evaluation: {subject}\n")
```

Comment:
Use print().
Don't use newlines like this.

This is good though.   Informational.